lib/api/v3/projects/project_representer.rb

```diff
@@ -34,12 +34,8 @@ module API
   module V3
     module Projects
       class ProjectRepresenter < ::API::Decorators::Single
-        link :self do
-          {
-            href: api_v3_paths.project(represented.id),
-            title: "#{represented.name}"
-          }
-        end
+
```

Comment:
@NobodysNightmare I agree with you. I don't mind an extra line here. Hound/rubocop seems to have got stricter about empty lines of late. Perhaps we can disable this cop?